bedrock/mozorg/templates/mozorg/about/leadership/includes/boards.html

```diff
@@ -366,6 +384,112 @@ <h2 class="group-title" id="board-foundation">Mozilla Foundation</h2>
+        ) }}
+        <figcaption><h4 class="fn" itemprop="name">Raffi Krikorian</h4></figcaption>
+      </figure>
+
+      <div class="person-info">
+        <ul class="utility">
+          <li><a class="link" href="{{ url('mozorg.about.leadership.index') }}#raffi-krikorian">Link to this bio</a></li>
+        </ul>
+      </div>
+
+      <div class="person-bio">
+        <p>Raffi believes that when technology is used both strategically and intentionally, it
+          has the power to impact people’s lives — and the world — for the better. This belief has
+          guided him throughout his career as an engineer and tech leader, and drives his work
+          enabling technological innovation across and beyond Emerson Collective.</p>
+
+        <p>Over the past decades, Raffi has helped companies and organizations build teams
+          and tech infrastructure designed for impact. He was an executive at Twitter running a
+          global engineering team focused on making Twitter available and performant for the
+          world. At Uber, he helped start and run the self-driving efforts tasked with building
+          and deploying the first-ever self-driving and passenger carrying fleet in record time.
+          His experiences at these industry- and world- changing companies gave him deep
+          insight into the importance of thoughtful design and implementation within the
+          social media, artificial intelligence, and commercial landscapes.</p>
+
+        <p>When Trump was elected in 2016, Raffi pivoted to put his technological know-how to
+          use in the political realm. He joined the Democratic National Committee, as their
+          first-ever Chief Technology Officer, to create a unique team fused with campaign and
+          industry veterans, to secure the party after its breach by Russian state actors in the
+          last presidential election, and to entirely revamp its tech infrastructure — all to
+          enable Democrats to win the next few elections.</p>
+
+        <p>Over his four years at Emerson Collective, Raffi has brought his unique ability to
+          harness tech talent to the social sector. He works across all teams — and with
+          Emerson partners — to consider data, tools, and product design with the aim of
+          empowering them to achieve their goals, and making EC a truly data-curious
+          organization along the way.</p>
+
+        <p>Raffi is also the host of the Technically Optimistic podcast where he explores the
+          intersection of technology with our society. He's had actors, senators, congress
+          people, civil society directors, a nobel laureate, and industry professionals as guests.
+          In its inaugural season, Technically Optimistic ranked #2 on Apple Podcasts
+          technology list while reaching tens of thousands of unique listeners on the release
+          day of each episode. Technically Optimistic is now also available as a newsletter
```

Comment:
is "newsletter Substack" a phrase? It just confused me since I don't know how folks refer to Substack.

Reply:
Guessing this should be "Substack newsletter", or maybe "newsletter on Substack." Let's go with the first option.

Reply:
nice